app/models/ccms/submission.rb

```diff
@@ -0,0 +1,140 @@
+module CCMS
+  class Submission < ApplicationRecord # rubocop:disable Metrics/ClassLength
```

Comment:
This model is too big. There are two steps that should be made to reduce it's size:

1. Strip the state machine functionality out into a concern (see `LegalAidApplicationStateMachine`
2. Split the complicated methods into services.